charts-other/sidam/charts/sidam-fr-idm/templates/service.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-{{ .Chart.Name }}
+  labels:
+    app.kubernetes.io/name: {{ .Release.Name }}-{{ .Chart.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}-{{ .Chart.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+spec:
+  selector:
+    app.kubernetes.io/name: {{ .Release.Name }}-{{ .Chart.Name }}
+  ports:
+    - name: jxm
+      port: 9010
+      targetPort: 9010
+    - name: web
+      port: 18080
+      targetPort: 18080
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: fr-idm
+  labels:
+    app.kubernetes.io/name: {{ .Release.Name }}-{{ .Chart.Name }}
```

Comment:
Hmm, don't understand why this service duplication is required? 🤔

Reply:
This is funny bit. ForgeRock images are not parametrised and they have some host names hardcoded so nothing other that fr-idm won't work. In final version I will keep just one version. I started my own naming convention but later I learnt that it cannot be used because of these hardcoded host names.